docs-chef-io/content/server/upgrades.md

```diff
@@ -106,6 +106,11 @@ If you are running a Chef Infra Server release before 12.3.0, please contact Che
 
 ## Chef Infra Server 14 Upgrade Process
 
+### Frontends in a Chef Backend HA
+
+Skip the first two steps in the Standalone Server section, then proceed to install the package and upgrade. Also skip the final reindex step.
+Anything to do with the database in a Chef Backend cluster will happen on the backend cluster.
```

Comment:
```suggestion
{{< note >}}
For frontend nodes in a Chef Backend HA cluster, skip the first two steps in the [Standalone Server](#standalone-server) section.
{{< /note >}}
```

Reply:
It is useful to clarify `Anything to do with the database in a Chef Backend cluster will happen on the backend cluster.` explicitly.